### DISCUSSION TITLE
Safe Exam Browser: URL issue on Mac

### DISCUSSION THREAD

**Discussion by LucaKrenger** [SafeExamBrowser/seb-mac#203] [Q&A] [answered]
We tried to set up an exam in SEB and it does not work on Mac-PCs (we tried it on 3 different Macs). SEB can be started, but afterwards the screen is loading infinitely and stays blank. We tried it with different configurations and start URLs but it did not work. 

Do you have any suggestions, why this happened?

Thank you for your response.

**Comment by danschlet** [comment, marked as answer]
Did you enter **full URLs** (including https://), as the Start URL field states? 

If you did and it still doesn't work the send us ALL the information required for us to be able to analyze a problem, as listed on the page https://safeexambrowser.org/support_en.html.
The basic information we ALWAYS need and you didn't provide is:
- SEB version(s) used
- macOS version used
- Example of a not working SEB config file (remove passwords)
- SEB log files (depending on the issue reported)

**Reply by LucaKrenger** [reply]
Yes, we used the full URL, tried it with several different URLs and several different Mac PCs. The same files worked on windows and, with the new beta version, on iPad. 

- SEB 3.0 for Mac
- MAC: We used several different Macs, including version: macOS Catalina (10.15.7), macOS Monterey (12.0.1) and others.
- Config-File: [not working MAC-SEB-File.zip](https://github.com/SafeExamBrowser/seb-mac/files/8423950/not.working.MAC-SEB-File.zip)

**Reply by danschlet** [reply]
Safari on the latest iOS version actually also doesn't load your page, it shows an error that the server certificate isn't valid. SEB on macOS and iOS only works with 100% correct certificates (unless you use the setting to embed a custom SSL certificate). So if other browsers show warnings, SEB won't load the page. SEB for Windows might behave differently (as Apple seems to have stricter security policies for websites).

**Reply by LucaKrenger** [reply]
Sorry, my bad, I sent the wrong file: This one contains a working URL: 
[not working MAC-SEB-File.zip](https://github.com/SafeExamBrowser/seb-mac/files/8425880/not.working.MAC-SEB-File.zip)

**Reply by danschlet** [reply]
But this config file loads correctly for me, both with SEB 3.0 and SEB 3.1rc (the release candidate of the upcoming new version).

Make sure that you don't have the same kind of anti-malware software installed on all your Macs which would inspect https internet connections. Such software changes the root certificate to perform a man-in-the-middle attack on secured internet connections to be able to inspect transferred contents (and is an absolute no-go for anybody who understands something about internet security). SEB intentionally doesn't work with such so-called security software.

**Reply by LucaKrenger** [reply]
Thank you for your immediate response. We figured it out, the issue was related to the URL, which is now fine. Thanks for helping.

**Comment by tjpete** [comment]
This is not the same subject but relative to the .seb file.  I am trying to deploy using JAMF.  I will make a pkg of the browser and copy user templates.  I am just not sure where the .seb file should be pushed for an M1 Silicon Mac laptop.  Can you help me with that? If we need another discussion thread please let me know. Thanks,

**Reply by danschlet** [reply]
See the "deployment" section in the SEB manual: https://safeexambrowser.org/macosx/mac_usermanual_en.html#ConfigDeployment

If you have more questions about this, yes, please open a separate discussion thread.
